README.md

```diff
@@ -1,159 +1,45 @@
-<img src=".erb/img/erb-banner.svg" width="100%" />
+# Notes App
 
-<br>
+This project is a notes app built using React and Electron. It allows users to create, organize, and search through their notes efficiently. The app features a simple, user-friendly interface designed for productivity and note management.
 
-<p>
-  Electron React Boilerplate uses <a href="https://electron.atom.io/">Electron</a>, <a href="https://facebook.github.io/react/">React</a>, <a href="https://github.com/reactjs/react-router">React Router</a>, <a href="https://webpack.js.org/">Webpack</a> and <a href="https://www.npmjs.com/package/react-refresh">React Fast Refresh</a>.
-</p>
+## Features
 
-<br>
+- **Markdown-based Note-Taking**: Create, edit, and save notes using markdown.
+- **File Organization**: Organize notes into folders and subfolders.
+- **Search Functionality**: Quickly search through notes.
 
-<div align="center">
+## Prerequisites
 
-[![Build Status][github-actions-status]][github-actions-url]
-[![Github Tag][github-tag-image]][github-tag-url]
-[![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/Fjy3vfgy5q)
+Before starting, ensure that you have the following tools installed:
 
-[![OpenCollective](https://opencollective.com/electron-react-boilerplate-594/backers/badge.svg)](#backers)
-[![OpenCollective](https://opencollective.com/electron-react-boilerplate-594/sponsors/badge.svg)](#sponsors)
-[![StackOverflow][stackoverflow-img]][stackoverflow-url]
+1. **Node.js** (v14 or later)
+2. **npm** or **yarn**
+3. **Git**
 
-</div>
+## Clone the Repository
 
-## Install
-
-Clone the repo and install dependencies:
+First, clone the repository and install the necessary dependencies:
 
 ```bash
 git clone --depth 1 --branch main https://github.com/electron-react-boilerplate/electron-react-boilerplate.git your-project-name
 cd your-project-name
 npm install
 ```
```

Comment:
This needs to be updated. The repo is pointing to his GitHub. It should be ours.

Reply:
fixed this one